h/static/scripts/test/widget-controller-test.coffee

```diff
@@ -1,16 +1,20 @@
 {module, inject} = angular.mock
 
+events = require('../events')
+
 describe 'WidgetController', ->
   $scope = null
   fakeAnnotationMapper = null
   fakeAnnotationUI = null
   fakeAuth = null
   fakeCrossFrame = null
+  fakeDrafts = null
   fakeStore = null
   fakeStreamer = null
   fakeStreamFilter = null
   fakeThreading = null
   fakeGroups = null
+  lastSearchResult = null
```

Comment:
This is only used by one test, does it need to be at such a high scope? I think a lot of our tests (especially client-side ones) are pretty hard to understand because to understand a given test method I need to understand not just that test method but lots of other bits of code spread throughout the test module that it depends on. And sometimes these shared fake objects that are shared between tests include fairly detailed fake behaviour that couples tests together, since changing the fake can break many tests at once.

If just one test needs to save the last search result it might be clearer if that test included the code for doing that, rather than adding it to the shared fake.